cmd/bootstrap/cmd/constraints.go

```diff
@@ -24,40 +24,10 @@ func ensureUniformNodeWeightsPerRole(allNodes flow.IdentityList) {
 	}
 }
 
-// Checks constraints about the number of partner and internal nodes.
-//   - Internal nodes must comprise >2/3 of each collector cluster.
+// Checks constraints:
 //   - for all roles R:
 //     all node with role R must have the same weight
-func checkConstraints(partnerNodes, internalNodes []model.NodeInfo) {
-	partners := model.ToIdentityList(partnerNodes)
-	internals := model.ToIdentityList(internalNodes)
-	all := append(partners, internals...)
-
+func checkConstraints(stakingNodes []model.NodeInfo) {
```

Comment:
This could be renamed to `CheckWeightsPerRole` but I left it in case we want to check other types of constraints within the same function.